src/app/header/header.component.html

```diff
@@ -7,5 +8,10 @@
     <div class="col-md-10 text-center">
         <span class="header-white-1 text-center">{{language?.headings.titleLeft}}</span><span class="header-2 text-center">{{language?.headings.titleRight}}</span>
     </div>
-    <button type="button" class="btn btn-default logout-button" aria-label="Left Align" (click)="logout()">{{language?.messages.logoutButton}}</button>
+  <div class="icons">
+    <i matTooltip="Update Profile" class="fa fa-pencil-square-o"></i>&nbsp; &nbsp;
```

Comment:
add the onclick listener for this as well, since the implementation is there in the other PR, we will have the functionality when the other PR is merged and we can avoid revisiting this again later.

Reply:
Ya! But @Raxerz this would require importing the update component which is not currently in this PR. So I guess I add this button in the update component and then later we would deal with merge conflicts.